designer/client/src/components/history/HistoryItem.tsx

```diff
@@ -42,17 +29,15 @@ const HDate = ({ date }: { date: string }) => (
 export function HistoryItem({ onClick, version, type, isLatest, isDeployed }: HistoryItemProps): JSX.Element {
     const { t } = useTranslation();
     const { user, createDate, processVersionId, actions } = version;
-    const className = useMemo(() => mapVersionToClassName(type), [type]);
 
-    const flex = css({ display: "flex", justifyContent: "space-between", alignItems: "flex-start" });
-    const badgeStyles = css({ height: "1.2em", margin: "0 1.2em" });
     return (
-        <li className={cx(className, flex)} onClick={() => onClick(version)}>
+        <HistoryItemStyled type={type} onClick={() => onClick(version)}>
```

Comment:
It's a problem with styles here:

Staging
<img width="299" alt="image" src="https://github.com/TouK/nussknacker/assets/9945753/916bd447-5f34-4a16-88d8-50f71ccada39">

locally after these changes:

<img width="292" alt="image" src="https://github.com/TouK/nussknacker/assets/9945753/07f97fa2-6d13-4d28-a665-a21909ff77dd">

1. Text color based on state
2. There shouldn't be a hover effect on active dot

Reply:
fix

Reply:
Nice, one more thing, there is still a hover event on active items, on staging version there is no hover effect 
<img width="358" alt="image" src="https://github.com/TouK/nussknacker/assets/9945753/bd39e4a4-e1b1-4817-8d97-9f7a517e5a98">